nncf/common/pruning/schedulers.py

```diff
@@ -190,17 +198,18 @@ def _init_exp(epoch_idx, p_min, p_max, factor=0.125):
         :param p_max: Target pruning level at which the schedule ends.
         :param factor: Hyperparameter.
         """
-        def get_b(a, k):
+        def get_b(a):
             return p_min - a
 
         def get_a(k):
-            return (3 / 4 * p_max - p_min) / (np.exp(-k * factor * epoch_idx) - 1)
+            return (p_max - p_min) / (np.exp(-k * epoch_idx) - 1)
 
         def f_to_solve(x):
-            y = np.exp(factor * x * epoch_idx)
-            return 1 / 3 * y + 1 / (y ** 7) - 4 / 3
+            c = (0.75 * p_max - p_min) / (p_max - p_min)
+            y = np.exp(-x * epoch_idx)
+            return y ** factor - c * y + c - 1
```

Comment:
Should we make the `factor` parameter adjustable via config? I suppose the 1/8 value is based on what is set in the paper, not on our empirical results.

Reply:
Yes, it is a good idea to set better default value based on empirical results on our models (firstly get these results). But I am not sure about the importance of this param in the config for users.